setup.py

```diff
@@ -7,15 +8,87 @@ def readme():
     return content
 
 
-version_file = 'mim/version.py'
-
-
 def get_version():
+    version_file = 'mim/version.py'
     with open(version_file) as f:
         exec(compile(f.read(), version_file, 'exec'))
     return locals()['__version__']
 
 
+def parse_requirements(fname='requirements/runtime.txt', with_version=True):
+    """Parse the package dependencies listed in a requirements file but strips
+    specific versioning information.
+
+    Args:
+        fname (str): path to requirements file
+        with_version (bool, default=False): if True include version specs
+
+    Returns:
+        List[str]: list of requirements items
+
+    CommandLine:
+        python -c "import setup; print(setup.parse_requirements())"
+    """
```

Comment:
This function does not support parsing the complete [Requirements File Format](https://pip.pypa.io/en/stable/reference/requirements-file-format/), but only handles part of the options. This function can be improved in the future. For example, ignore unknown options

Reply:
OK